azerothcore/mod-guildhouse
# GH-Faction-Neutral-Food-Vendor

## Changes Proposed:
- Food/Drink Vendor is currently set to Brogus Thunderbrew, which is hostile to horde player, proposed to update the database and code to use "Gant" in Area52, which is a faction neutral vendor
- 

## Issues Addressed:

https://github.com/azerothcore/mod-guildhouse/issues/36
- Closes


## Tests Performed:
-  Build and tested on latest build
- 


## How to Test the Changes:


1.  (if GH food vendor already exists, select and run `.npc delete`
2.  Go back to GH butler, purchase food/drink vendor
3.  Should see Gant instead of Brogus.  Gant is neutral to both horde and alliance.
